plugins/ksp/src/org/jetbrains/kotlin/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -65,6 +68,12 @@ class ResolverImpl(
     val javaActualAnnotationArgumentExtractor = JavaActualAnnotationArgumentExtractor()
     private val nameToKSMap: MutableMap<KSName, KSClassDeclaration>
 
+    private val typeMapper = KotlinTypeMapper(
```

Comment:
I'm a bit worried if `KotlinTypeMapper` might be getting deprecated. Could we check with upstream?